source/containers.tex

```diff
@@ -2888,18 +2889,18 @@
   class deque {
   public:
     // types:
-    typedef value_type&                                         reference;
-    typedef const value_type&                                   const_reference;
-    typedef @\impdefnc@                              iterator;        // see \ref{container.requirements}
-    typedef @\impdefnc@                              const_iterator;  // see \ref{container.requirements}
-    typedef @\impdefnc@                              size_type;       // see \ref{container.requirements}
-    typedef @\impdefnc@                              difference_type; // see \ref{container.requirements}
-    typedef T                                                   value_type;
-    typedef Allocator                                           allocator_type;
-    typedef typename allocator_traits<Allocator>::pointer       pointer;
-    typedef typename allocator_traits<Allocator>::const_pointer const_pointer;
-    typedef std::reverse_iterator<iterator>                     reverse_iterator;
-    typedef std::reverse_iterator<const_iterator>               const_reverse_iterator;
+    using value_type             = T;
+    using allocator_type         = Allocator;
+    using pointer                = typename allocator_traits<Allocator>::pointer;
+    using const_pointer          = typename allocator_traits<Allocator>::const_pointer;
+    using reference              = value_type&;
+    using const_reference        = const value_type&;
+    using size_type              = @\impdefnc@; // see \ref{container.requirements}
```

Comment:
Correct. "nc" stands for "no (italic) correction", which affects the column alignment. You ought to be able to see the difference (look for how characters (fail to) align on the grid without the correction). But now that the relevant text is at the end of the line, there's nothing left to be aligned, and we don't need to control the italic correction.

That was one of the benefits of changing `typedef` to `using` that @zygoloid pointed out in his first email about this subject a few months ago if I recall correctly.